tests/core/models/test_edition.py

```diff
@@ -325,6 +328,67 @@ def test_calculate_presentation_author(self, db: DatabaseTransactionFixture):
         assert "Kelly Accumulator, Bob A. Bitshifter" == wr.author
         assert "Accumulator, Kelly ; Bitshifter, Bob" == wr.sort_author
 
+    def test_calculate_presentation_very_long_author(
+        self, db: DatabaseTransactionFixture
+    ):
+        authors = []
+
+        # author names should be unique and not similar to ensure that the
+        # test mirrors the types of long author lists we'd expect in real data.
+        def generate_random_random_author():
```

Comment:
Did you mean "...random_random..." in this function name?

Reply:
No - I'll fix.